hamcrest/Hamcrest/File/FileMatcher.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @author Vasek Brychta <vaclav@brychtovi.cz>
+ */
+
+namespace Hamcrest\File;
+
+use Hamcrest\BaseMatcher;
+use Hamcrest\Description;
+
+abstract class FileMatcher extends BaseMatcher
+{
+	/** @var string */
+	private $ownDescription;
+	/** @var string */
+	private $failureDescription;
```

Comment:
Please follow the code format (also in comments) of the main codebase.

P.S.
The same applies to other places of the code.

---

DocBlock style doesn't match the rest of the matchers. They either don't have DocBlocks or format them in a multi-line way.

Reply:
I'm sorry, I was too focused on the functionality itself that I forgot to maintain the style. Is there any cookbook/style definition or should I just try to find a similar case in the codebase?

Reply:
I have no idea. The https://github.com/hamcrest/hamcrest-php/blob/master/CONTRIBUTING.md file mentions coding standard but doesn't tell what it is. I'm assuming it's PSR-2.

Anyway it's usually best to copy/paste new files from parts of existing file to ensure that code style is the same.